src/demos/modules/script-tag.html

```diff
@@ -0,0 +1,44 @@
+---
+title: Module and Script
+
+eleventyNavigation:
+    key: Module and Script
```

Comment:
Here the key is "Module and Script". So I guess it doesn't really matter as long as Key is a unique string. But maybe we should standardize to a camelCaseKey.